dagger-1-migration.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+title: Migrating from Dagger 1
+---
+
+While Dagger 1 and Dagger 2 are similar in many ways, one is not a drop-in replacement for the other.  This guide will highlight both the API and conceptual differences between the two versions and provide recommended patterns for migrating.
+
+## Injected types
+
+Dagger 2 continues to rely on [JSR 330](https://jcp.org/en/jsr/detail?id=330) for declaring injection sites. All of the [types of injection](http://docs.oracle.com/javaee/6/api/javax/inject/Inject.html) supported by Dagger 1 (field and constructor) continue to be supported by Dagger 2, but Dagger 2 supports method injection as well. Dagger 2 ***does not*** support static injection.
+
+## Framework types
+
+Dagger 2 also continues to support both the JSR 330 and Dagger-specific framework types. [`Provider`][Provider], [`Lazy`][Lazy] and [`MembersInjector`][MembersInjector] all maintain the same behavior.
+
+## Modules
+
+Dagger 2 [modules][Module] are very similar to Dagger 1 modules, but reduce much of the configuration complexity that had been a pain point for some.  For most users, migration will simply be deleting properties.
+
+### Module Properties
+
+The primary differences between the two stem from the fact that _graph validation_ is no longer performed on a per-module basis. In Dagger 1 terms, Dagger 2 modules are all declared as `complete = false` and `library = true`, but since that is most permissive for validation those properties and `addsTo` can just be removed when migrating.
```

Comment:
...=true` but, since that is the most permissive for validation, those...

"since that is the most permissive validation" is the explanatory sub-phrase.

And to make it less ambiguous, I'd say "...validation, `library`, `complete`, and `addsTo` can simply be removed...".  As in, just name all three in what can be removed.